test/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+# Project information.
+cmake_minimum_required ( VERSION 3.13.0 )
+project ( "CoreHTTP unit test"
```

Comment:
hmm why did we choose to capitalize the C here when the repo is named with a lowercase c?

Reply:
Ill lowercase thanks